core/src/main/java/org/mobilitydata/gtfsvalidator/annotation/FieldType.java

```diff
@@ -0,0 +1,15 @@
+package org.mobilitydata.gtfsvalidator.annotation;
+
+import java.lang.annotation.ElementType;
+import java.lang.annotation.Retention;
+import java.lang.annotation.RetentionPolicy;
+import java.lang.annotation.Target;
+
+/**
+ * Specifies type of a GTFS field, e.g., {@code COLOR} or {@code LATITUDE}.
```

Comment:
What do you think about saying "{@code COLOR}, {@code LATITUDE}, etc", to explicitly show that there are more than just two field types? Also, could you add an expectation of how this annotation is going to be used? In other words, as a developer I'd like to know answer to questions such as
 * Do I have to specify this annotation?
 * What benefits will I get by specifying this notification?
 * What Java data types must I use with a particular FieldType (if any)? I think, FieldTypeEnum should clarify this per field type and here we should say that there's such limitation are applied. Just to add a bit more context, what can I use with FieldTypeEnum.INTEGER? 'short', 'int', 'long', BigInteger?

Reply:
Very good point. I added documentation.

I say "e.g, COLOR or LATITUDE", so it suggests that I am giving examples and not a full list :)